fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCParameterBuildingVisitor.java

```diff
@@ -432,12 +432,12 @@ public boolean visit(java.lang.String elementName, int elementIndex, Quantity qu
             // see https://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=19597
             if (quantity.getCode() != null) {
                 p.setValueCode(quantity.getCode().getValue());
+                if (quantity.getSystem() != null) {
+                    p.setValueSystem(quantity.getSystem().getValue());
```

Comment:
Looking at this branching it occurs to me... 
should Parameter object have a default getSystem ?  

How do we handle null? sorry if I am re-treading

Reply:
the DAO layer inserts a default system if its null, but my understanding is its just a database perf thing.  a quantity SHOULD have a code and a system, but isn't required to.